src/Graphics-Canvas/BitmapFillStyle.class.st

```diff
@@ -34,7 +34,7 @@ BitmapFillStyle class >> fromForm: aForm [
 { #category : #'instance creation' }
 BitmapFillStyle class >> fromUser [
 	| fill |
-	fill := self form: Form fromUser.
+	fill := self form: Screenshot new formScreenshotFromUserSelection.
```

Comment:
#formScreenshotFromUserSelection is not a good selector
- "screenshot" in the selector is redundant as the receiver is already Screenshot (instance)
- having the type of the result in the selectors name is also not good as other types could be returned as long as they work polymorphic

I would do it with a more convinient method:

"Screenshot fromUserSelection" which returns the form